modules/wpcom-block-editor/functions.classic-editor.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * This file contains some 'remember' functions taken from the core Classic Editor Plugin
+ * Used to align the 'last editor' metadata so that it is set on all Jetpack and WPCOM sites
+ *
+ * @package Jetpack
+ */
+
+namespace Jetpack\ClassicEditor;
+
+/**
+ * Remember when the classic editor was used to edit a post.
+ *
+ * @param object $post The post being editted.
+ */
+function remember_classic_editor( $post ) {
+	$post_type = get_post_type( $post );
+
+	if ( $post_type && post_type_supports( $post_type, 'editor' ) ) {
+		remember_editor( $post->ID, 'classic-editor' );
+	}
+}
+
+/**
+ * Remember when the block editor was used to edit a post.
+ *
+ * @param  object $editor_settings This is hooked into a filter and this is the settings that are passed in.
+ * @param  object $post            The post being editted.
+ * @return object                  The unmodified $editor_settings parameter.
+ */
+function remember_block_editor( $editor_settings, $post ) {
+	$post_type = get_post_type( $post );
+
+	if ( $post_type && classic_editor_can_edit_post_type( $post_type ) ) {
+		remember_editor( $post->ID, 'block-editor' );
+	}
+
+	return $editor_settings;
+}
+
+/**
+ * Sets the metadata for the specified post and editor.
+ *
+ * @param int    $post_id The ID of the post to set the metadata for.
+ * @param string $editor  String name of the editor, 'classic' or 'gutenberg'.
+ */
+function remember_editor( $post_id, $editor ) {
+	if ( get_post_meta( $post_id, 'classic-editor-remember', true ) !== $editor ) {
+		update_post_meta( $post_id, 'classic-editor-remember', $editor );
```

Comment:
Do we have the option to update that post meta? That name makes it seem like we'd expect a boolean here, and since it doesn't have an underscore as a prefix, it's going to be displayed in the custom fields interface:

![image](https://user-images.githubusercontent.com/426388/95741321-5d970600-0c8e-11eb-8082-ba954210f991.png)

Maybe we could name this something like `_jetpack_post_editor`?

```suggestion
	if ( get_post_meta( $post_id, '_jetpack_post_editor', true ) !== $editor ) {
		update_post_meta( $post_id, '_jetpack_post_editor', $editor );
```

Reply:
Yes, I didn't think it was the best name but I was using the key that is being used by the Classic Editor Plugin. Making the metadata hidden is reason to update it, so we may as well improve the name too. 

Should we perhaps set this regardless of whether the Classic Editor Plugin is active?

Reply:
Ah right. I suppose we should stick to the standard set by the classic editor plugin.

Reply:
Yeah maybe! I'll open an issue/PR on the Classic Editor Plugin to see if we can change it to something like `_last_editor_used`